cves/kernel/CVE-2020-12770.yml

```diff
@@ -337,9 +363,10 @@ stacktrace:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: false
+  stacktrace_with_fix: false
+  note: |
```

Comment:
This note can probably go to one line ! In mine I only used the multi-line delimiter when I had two or more lines

Reply:
That would make it look a bit cleaner. Thanks for the suggestion.